defaultConfig/addMissingDependencies.config.json

```diff
@@ -182,6 +182,19 @@
 				"extender": "AddUnusedImportWithComment",
 				"version": "^1.58.0"
 			}
+		},
+		"Replacement interim solution fix": {
+			"UriParameters": {
+				"newExpressionArgsLength": 1,
+				"newExpressionCalleeName": "UriParameters",
+				"newVariableName": "UriParameters",
+				"functionName": "fromURL",
```

Comment:
when we generically use fromURL as replacement, then the change for UriParameters was useless. The idea was to differentiate between the fromURL and fromQuery calls. 

Please check the migration documentation for UriParameters (class documentation).

Reply:
@codeworrior I would keep the replacement simple.
from a code perspective in jquery.sap.script.js:
```js
jQuery.sap.getUriParameters(sUri)
```
becomes
```js
UriParameters.fromURL(sUri || window.location.href)
```

Within the "Replacement interim solution fix" (
80bc977) in addMissingDependencies the only findings can be either:
```js
new UriParameters(sUri || window.location.href)
```
or 
```js
new UriParameters(window.location.href)
```
therefore the replacement with UriParameters.fromURL is valid.

The only thing one can think of is in (b7aa35d) if we can identify a full url. E.g. if someone calls it with a string
jQuery.sap.getUriParameters("http://....") where we can separate fromQuery and fromURL